api/k8sconsts/odiglet.go

```diff
@@ -11,4 +11,7 @@ const (
 	OdigletContainerName          = "odiglet"
 	OdigletImageName              = "keyval/odigos-odiglet"
 	OdigletEnterpriseImageName    = "keyval/odigos-enterprise-odiglet"
+	// Used to indicate that the odiglet is installed on a node.
+	OdigletInstalledLabel      = "odiglet-installed"
```

Comment:
```suggestion
	OdigletInstalledLabel      = "odigos.io/odiglet-installed"
```

I think that's a better practice for naming kabels/annotations